tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc

```diff
@@ -470,6 +470,65 @@ void ral_qgemm(
   ctx->signalError(Context::FAILURE, "Should turn on bladnn first.");
 }
 
+template <int N>
+MemRefType<int8_t, N> ral_pdll_qgemm_s8s8s8_biasadd_quant_per_tensor(
```

Comment:
maybe we can add 'nt' tag in the kernel name since it supposes that the weight is transposed before.

Reply:
done